packages/common/src/compiler/filtersCompiler.test.ts

```diff
@@ -806,3 +812,197 @@ describe('Filter SQL', () => {
         ).toBe('1=1');
     });
 });
+
+describe('convertMetricFilterToDbt', () => {
+    it('should return undefined if filters are undefined', () => {
+        expect(convertMetricFilterToDbt(undefined)).toBeUndefined();
+    });
+
+    it('should return an empty array if filters are an empty array', () => {
+        expect(convertMetricFilterToDbt([])).toEqual([]);
+    });
+
+    it('should convert EQUALS filter correctly', () => {
+        const filters: MetricFilterRule[] = [
+            {
+                target: { fieldRef: 'customers.customer_id' },
+                id: '1',
+                operator: FilterOperator.EQUALS,
+                values: ['value1'],
+            },
+        ];
+        const expected: DbtColumnLightdashMetric['filters'] = [
+            { customer_id: 'value1' },
+        ];
+        expect(convertMetricFilterToDbt(filters)).toEqual(expected);
+    });
+
+    it('should convert NOT_EQUALS filter correctly', () => {
+        const filters: MetricFilterRule[] = [
+            {
+                target: { fieldRef: 'field2' },
+                id: '1',
+                operator: FilterOperator.NOT_EQUALS,
+                values: ['value2'],
+            },
+        ];
+        const expected: DbtColumnLightdashMetric['filters'] = [
+            { field2: '!value2' },
+        ];
+        expect(convertMetricFilterToDbt(filters)).toEqual(expected);
+    });
+    it('should convert NULL filters correctly', () => {
+        const filters: MetricFilterRule[] = [
+            {
+                target: { fieldRef: 'field2' },
+                id: '1',
+                operator: FilterOperator.EQUALS,
+                values: [null],
```

Comment:
Should we test the IS_NULL operator? I don't think we usually use EQUALS + null value

```
{
                "id": "15204703-20d0-4b20-9a3c-277963bc2356",
                "operator": "isNull",
                "values": [
                    1
                ],
                "target": {
                    "fieldRef": "events.event"
                }
            },
```